contacts/tasks.py

```diff
@@ -0,0 +1,17 @@
+from django.core.mail import EmailMessage
+from phone_book.celery import app
+from celery.utils.log import get_task_logger
+
+logger = get_task_logger(__name__)
+
+
+@app.task(name='send')
+def send(mail_subject, to_email, message):
+
+    email = EmailMessage(
+        mail_subject, message, to=[to_email]
+    )
+    email.send()
+    return "Email Has been sent from Task of sending email"
```

Comment:
I meant to say, you should include the email of the person who this email was sent to.